docs/adr/0003-content-organization-and-accesses.rst

```diff
@@ -0,0 +1,43 @@
+ADR 0003 - Content organization and accesses
+============================================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+We have `actors <0001-actors>`_, and `videos <0002-videos-languages>`_, that we'll call for now "content". We want this content to be organized for actors to manage/view them.
+
+
+Decision
+--------
+
+Videos are grouped in playlists, which is a Django model named ``Playlist``. A playlist belongs to an organization (``Organization`` model defined in `actors <0001-actors>`_) and is created by someone who can be a manager of this organization, or an author who belongs to this organization. This link is materialized by an ``author`` field on the ``Video`` model, a ``ForeignKey`` to the ``User`` model.
+
+The manager can allow many authors to manage a playlist, so there is a ``ManyToMany`` field on the ``Video`` model, named ``editors``, pointing to the ``User`` model.
+
+The author of the playlist is automatically added to this list of editors. And can be removed from this list by a manager, still being named as the author of the playlist.
+
+A playlist can be duplicated by a manager, and if it stays in the same organization, the manager can clear or keep the list of editors. If it is to de duplicated in another organization, the list of editors will be cleared of authors not belonging to the new organization, and the manager will still be able to clear it all or keep the remaining editors.
+
+When duplicated, a new instance of ``Playlist`` is created, same for each instances of the ``Video`` belonging to this model, but we'll keep the same links to the files (videos/audios/subtitles) on the host provider, to keep cost manageable.
+
+And finally, there is a flag named ``public`` on the playlist, that can be toggled by a manager, to tell if the playlist can be viewed by anyone or only by people who were granted access to it. This kind of access is not in the scope of this document.
+
+Ongoing discussions
+-------------------
+
+1. Do we keep the author when duplicating a playlist? Or the manager can make the choice? I am in favor of keeping it.
```

Comment:
First point to discuss in addition to the `Decision` section

Reply:
+1 for keeping the author